source/agora/node/admin/AdminInterface.d

```diff
@@ -125,9 +125,6 @@ public class AdminInterface
 
     public void stop ()
     {
-        if (!this.config.admin.enabled)
```

Comment:
Also this check is already here: https://github.com/bpfkorea/agora/blob/9097aa132ec62022635ef06355fb7174d27e9f41/source/agora/node/Validator.d#L432

Reply:
Yep, there are a few other checks which could be simplified, but this one stood out.